### PR TITLE
WRN-12124: Remove deprecated babel-eslint module

### PR DESCRIPTION
**Pull request by hong6316**
### Issue Resolved / Feature Added
`babel-eslint` has been deprecated

```
`babel-eslint` is now `@babel/eslint-parser`. This package will no longer receive updates.
```

### Resolution
Removed deprecated `babel-eslint`

### Additional Considerations
When you release the new @enact/cli version with this PR, you should update `npm-shrinkwrap.json`. 

> $ rm -rf npm-shrinkwrap.json
$ npm i
$ npm audit fix
$ npm shrinkwrap

### Links
WRN-12124

### Comments
Enact-DCO-1.0-Signed-off-by: Taeyoung Hong (taeyoung.hong@lge.com)

### CODE REVIEW

**Review by MikyungKim — COMMENTED** [review]
`babel-eslint` is remained in README.md. Please update that too.

**Comment by hong6316** [comment]
> `babel-eslint` is remained in README.md. Please update that too.

@MikyungKim 
I checked README.md and I remained this word first with in my opinion.
Because this description is the guide for removing previous modules that are no longer used.
(and customers will remove `babel-eslint` that we intended)

But anyway, I remove the word `babel-eslint` in README.md.

File: README.md [hunk 1]
```diff
@@ -112,7 +112,7 @@ npm install -g eslint
 Then, you will need to uninstall any previous globally-installed Enact linting package (everything but eslint itself):
 
 ```sh
-npm remove -g eslint-plugin-react eslint-plugin-react-hooks eslint-plugin-babel babel-eslint eslint-plugin-jest eslint-plugin-enact eslint-config-enact
+npm remove -g eslint-plugin-react eslint-plugin-react-hooks eslint-plugin-babel @babel/eslint-parser eslint-plugin-jest eslint-plugin-enact eslint-config-enact
```
Comment (MikyungKim) [inline README.md:115]: Here also, don't we need to add `babel-preset-react-app` or not?

**Review by MikyungKim — COMMENTED** [review]
Please add a change log.

File: package.json [hunk 2]
```diff
@@ -40,6 +40,7 @@
   ],
   "dependencies": {
     "@babel/core": "^7.16.7",
+    "@babel/eslint-parser": "^7.16.3",
```
Comment (MikyungKim) [inline package.json:43]:
```suggestion
    "@babel/eslint-parser": "^7.17.0",
```

File: README.md [hunk 3]
```diff
@@ -112,7 +112,7 @@ npm install -g eslint
 Then, you will need to uninstall any previous globally-installed Enact linting package (everything but eslint itself):
 
 ```sh
-npm remove -g eslint-plugin-react eslint-plugin-react-hooks eslint-plugin-babel babel-eslint eslint-plugin-jest eslint-plugin-enact eslint-config-enact
+npm remove -g eslint-plugin-react eslint-plugin-react-hooks eslint-plugin-babel @babel/eslint-parser babel-preset-react-app eslint-plugin-jest eslint-plugin-enact eslint-config-enact
```
Comment (MikyungKim) [inline README.md:115]:
```suggestion
npm remove -g eslint-plugin-react eslint-plugin-react-hooks eslint-plugin-babel @babel/eslint-parser eslint-plugin-jest eslint-plugin-enact eslint-config-enact
```

**Review by MikyungKim — APPROVED** [review]
LGTM

**Pull request merged**